reviews/models/mixins.py

```diff
@@ -0,0 +1,60 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from transitions import Machine
+
+from django.db import models
+
+from reviews import workflow
+
+
+class ReviewProviderMixin(models.Model):
+    """A reviewed/moderated collection of objects.
+    """
+
+    review_workflow = models.CharField(max_length=15, choices=workflow.WORKFLOW_CHOICES, default=workflow.PRE_MODERATION)
+    review_comments_public = models.BooleanField(default=False)
+    review_comments_anonymous = models.BooleanField(default=True)
+
+    class Meta:
+        abstract = True
+
+
+class ReviewableMixin(models.Model):
+    """Something that may be included in a reviewed collection and is subject to a reviews workflow
+    """
+
+    state = models.CharField(max_length=15, db_index=True, choices=workflow.STATE_CHOICES, default=workflow.PENDING)
```

Comment:
It would have been, but transitions is hardcoded to use `state`. I agree, though, it needs to be prefixed... we could use a machine that doesn't wrap the model, and instead run/validate transitions in `pre_save`, maybe?